frontend/src/metabase-types/api/settings.ts

```diff
@@ -32,6 +32,11 @@ export type LoadingMessage =
   | "running-query"
   | "loading-results";
 
+export type TokenStatus = {
+  status?: "unpaid" | "past-due" | string;
```

Comment:
Please extract to a separate type, e.g. `export type TokenStatusType = "unpaid" | ...`